platform/platform-filter-response/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -25,6 +25,9 @@
         <AD description="The HTTP X-XSS-Protection response header is a feature that stops pages from loading when they detect reflected cross-site scripting (XSS) attacks."
             name="X-XSS-Protection" id="xXssProtection" required="true" type="String"
             default="1; mode=block"/>
+        <AD description="The Cache-Control header specifies the behavior for browsers and proxies to cache web resources."
+            name="Cache-Control" id="cacheControl" required="true" type="String"
+            default="Cache-Control: \"public, max-age=604800, immutable\""/>
```

Comment:
I'm going to guess "private". @stustison 

_public
Indicates that the response MAY be cached by any cache, even if it would normally be non-cacheable or cacheable only within a non- shared cache. (See also Authorization, section 14.8, for additional details.)
private
Indicates that all or part of the response message is intended for a single user and MUST NOT be cached by a shared cache. This allows an origin server to state that the specified parts of the
response are intended for only one user and are not a valid response for requests by other users. A private (non-shared) cache MAY cache the response._